roofit/histfactory/inc/RooStats/HistFactory/HistoToWorkspaceFactoryFast.h

```diff
@@ -11,45 +11,49 @@
 #ifndef ROOSTATS_HISTOTOWORKSPACEFACTORYFAST
 #define ROOSTATS_HISTOTOWORKSPACEFACTORYFAST
 
-#include <vector>
-#include <string>
-#include <map>
-#include <iostream>
-#include <sstream>
-#include <memory>
+#include <RooStats/HistFactory/Systematics.h>
 
-#include <RooPlot.h>
 #include <RooArgSet.h>
+#include <RooDataSet.h>
 #include <RooFitResult.h>
-#include <RooAbsReal.h>
-#include <RooRealVar.h>
+#include <RooPlot.h>
 #include <RooWorkspace.h>
-#include <TObject.h>
+
 #include <TH1.h>
-#include <TDirectory.h>
+#include <TObject.h>
+
+#include <vector>
+#include <string>
+#include <map>
+#include <memory>
+
 
-#include "RooStats/HistFactory/Systematics.h"
 class ParamHistFunc;
 class RooProduct;
 class RooHistFunc;
 
-namespace RooStats{
-  namespace HistFactory{
-
-    // Forward Declarations FTW
-    class Measurement;
-    class Channel;
-    class Sample;
-
-    class HistoToWorkspaceFactoryFast: public TObject {
-
-    public:
-
-      struct Configuration {
-        bool binnedFitOptimization = true;
-        bool createPerRegionWorkspaces = true;
-        bool storeDataError = false;
-      };
+namespace RooStats {
+namespace HistFactory {
+
+// Forward Declarations FTW
+class Measurement;
+class Channel;
+class Sample;
+
+class HistoToWorkspaceFactoryFast : public TObject {
+
+public:
+   /// \brief Configuration settings for HistFactory behavior.
+   struct Configuration {
+      /// \brief Enable or disable optimization for binned likelihood fits (default `true`).
```

Comment:
By the way, if you want to save a few characters per docstring, our doxygen is set to take the first sentence up to the `.` as `\brief` if you don't specify any (`AUTOBRIEF`).

Reply:
Thanks! I'll keep this in mind for next time